ai-server/app/ml/train_model.py

```
@@ -0,0 +1,6 @@
+from sklearn.ensemble import RandomForestClassifier
+import joblib
+
+model = RandomForestClassifier()
+model.fit(X_train, y_train)
```

Ruff (0.12.2)

Undefined name `X_train`

(F821)